[BE] Adding user id in JWT payload

MihaiDruta opened the pull request:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

Commit:
[BE] Config default route for user details
https://trello.com/c/mB3WDYwH/70-be-config-default-route-for-user-details

If id in route is missing, you should be able to see the details of your user (authenticated user).

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

SpartakusMd commented on config/services.yaml:
```diff
@@ -7,6 +7,11 @@ parameters:
   locale: 'en'
 
 services:
+  acme_api.event.jwt_created_listener:
```
Rename to `api.event.jwt_created_listener` and move the section to the end of the file.

SpartakusMd commented on src/JWTListener/JWTCreatedListener.php:
```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\JWTListener;
```
Use `EventListener` instead of `JWTListener`. Rename the folder as well.

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload

Commit:
[BE] Adding user id in JWT payload
https://trello.com/c/mB3WDYwH/70-be-adding-user-id-in-jwt-payload

Added user id in JWT payload
